app.js

```diff
@@ -40,6 +41,7 @@ app.use(morgan(logger.morganConfiguration));
 app.use('/ping', ping);
 app.use('/v1/account', account);
 app.use('/v1/system', system);
+app.use('/v1/utils', utils);
```

Comment:
I'm not too keen on the name `utils` for the route, I think it should be something more descriptive. Maybe `external` and have this as a place to proxy through any other external API requests in the future. Or alternatively just `price`?

Reply:
```suggestion
app.use('/v1/external', external);
```
Seems go to me!